src/sentry/static/sentry/app/views/organizationRepositories.jsx

```diff
@@ -204,7 +204,7 @@ const OrganizationRepositories = React.createClass({
     return {
       loading: true,
       error: false,
-      itemList: null,
+      itemList: [],
```

Comment:
Yes, I definitely missed those;  rather than changing the initial state, I think it'll be better to fix the conditions in `render`